### PR TITLE
[ADD] studio: documentation export/import customizations

### PR DESCRIPTION
**Pull request by kbr-odoo**

### CODE REVIEW

File: studio/use_cases/export_import.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+=========================
+Export and Import Modules
+=========================
+
+When you do customizations in Odoo Studio, a new module is created in your database, making it easy
+to use Studio for prototyping.
+
+To export these customizations, on the main dashboard, activate Studio, and under the menu
+*Customizations*, click on *Export*. The default filename is *customizations.zip*.
```
Comment (Lau-Odoo) [inline studio/use_cases/export_import.rst:9]:
```suggestion
To export these customizations, activate Studio on the main dashboard and, under the menu
*Customizations*, click on *Export*. The default filename is *customizations.zip*.
```

File: studio/use_cases/export_import.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+.. image:: media//export_import/customizations_menu.png
+   :align: center
+   :alt: View of the menu customizations in Odoo Studio
+
+| The module created contains the definition of custom models and fields, as well as the UI elements
+  of any customization in an XML format.
```
Comment (Lau-Odoo) [inline studio/use_cases/export_import.rst:16]:
```suggestion
| The created module contains the definition of custom models and fields, as well as the UI elements
  of any customization in an XML format.
```

File: studio/use_cases/export_import.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+| To import and install the customizations in another instance, connect to the destination database,
+  activate Studio, and under *Customizations* click on *Import*.
```
Comment (Lau-Odoo) [inline studio/use_cases/export_import.rst:18]:
```suggestion
  activate Studio, and, under *Customizations*, click on *Import*.
```

File: studio/use_cases/export_import.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+.. image:: media/export_import/import_modules.png
+   :align: center
+   :alt: View of the import modules window for Odoo Studio
+
+.. warning::
+   Studio does not know which apps are customized (because the same view can be modified on
+   different apps), therefore, it *does not* add the underlying modules as dependencies of the
+   exported module. In other words, the applications installed on the source database should be
```
Comment (Lau-Odoo) [inline studio/use_cases/export_import.rst:27]:
```suggestion
   exported one. In other words, the applications installed on the source database should be
```

**Comment by kbr-odoo** [comment]
@Lau-Odoo thank you!

**Comment by jubodoo** [comment]
@kbr-odoo Just need to fix the conflicts

**Pull request merged**